.ci/scripts/distribution/random-test-java.sh

```diff
@@ -15,6 +15,10 @@ MAVEN_PROPERTIES=(
 )
 tmpfile=$(mktemp)
 
+export PROCESS_COUNT=10
+export EXECUTION_COUNT=100
+export REPLAY_EXECUTION_COUNT=5
+
```

Comment:
What do you think about moving that in to the maven profile https://github.com/camunda-cloud/zeebe/blob/develop/parent/pom.xml#L1381 ? https://stackoverflow.com/a/25605901/2165134

Reply:
It would make it more clear for what the properties are, otherwise I think we need a comment what the properties are actually for.

Reply:
I have moved the properties to the mentioned profile :)